source/WorkingPractices/testing.rst

```diff
@@ -71,26 +71,42 @@ change then this macro will be run by your code reviewer as part of the commit
 branch as described above and run the upgrade macro with one of the following
-commands
+commands, noting that ``--jules-path`` is only required if you have
+**jules-shared** metadata changes. Please see the shared metadata
+:ref:`guidance<shared-namelists>`.
 
-    +-------+----------------------------------------------------------------------------------------+
-    | UM    | $UMDIR/bin/update_all.py --path=/path/to/working/copy/of/test/branch --um=vnX.X_tXXXX  |
-    +-------+----------------------------------------------------------------------------------------+
-    | JULES | ./bin/upgrade_jules_test_apps vnX.X_tXXXX                                              |
-    +-------+----------------------------------------------------------------------------------------+
+    +-------+----------------------------------------------------------------------------------------------------------------------------------------------+
+    | UM    | ``$UMDIR/bin/update_all.py --path=/path/to/working/copy/of/test/branch --um=vnX.X_tXXXX --jules-path=/path/to/jules/working/copy/of/branch`` |
+    +-------+----------------------------------------------------------------------------------------------------------------------------------------------+
+    | JULES | ``./bin/upgrade_jules_test_apps vnX.X_tXXXX``                                                                                                |
+    +-------+----------------------------------------------------------------------------------------------------------------------------------------------+
 
 ..
-    | LFRic | *Macro capabilities are currently in development*                                      |
-    +-------+----------------------------------------------------------------------------------------+
+    | LFRic | *Macro capabilities are currently in development*                                                                                            |
+    +-------+----------------------------------------------------------------------------------------------------------------------------------------------+
 
 .. todo: add an LFRic upgrade macro command... and make these more consistent?
 
 .. Note::
-    The update_all.py script suppresses warnings produced by upgrade macros.
-    You can test these separately by upgrading a single app. A single app can be
-    upgraded for testing using ``rose app-upgrade -M /path/to/rose-meta -C /path/
-    to rose-stem/app/<app_name> -a <trunk_metadata_version>`` inside the app's
-    directory.
+   The update_all.py script suppresses warnings produced by upgrade macros.
+   You can test these separately by upgrading a single app. A single app can be
+   upgraded for testing using:
+
+   .. code-block::
+
+      rose app-upgrade -M /path/to/rose-meta -C /path/to/rose-stem/app/<app_name> -a <trunk_metadata_version>
+
+   where the ``-C`` option can be omitted if inside the app's
+   directory.
+
+   .. Important::
```

Comment:
I had added the important box so that it appeared within the note box as it applied to the instructions in the note box. If I move it above the note box, then it'll look like it applies to the update_all.py instructions, which it doesn't.